source/application/tasks/build-commonjs/index.js

```diff
@@ -58,7 +58,7 @@ async function exportAsCommonjs(application, settings) {
 	const cwd = application.runtime.patterncwd || application.runtime.cwd;
 
 	const patternRoot = resolve(cwd, 'patterns');
-	const commonjsRoot = resolve(cwd, 'build', 'build-commonjs');
+	const commonjsRoot = resolve(cwd, settings.out || 'build/build-commonjs');
```

Comment:
I will change this to using `path.join` for the fallback. However, node's path module in itself is already cross-platform and takes care of the slashes.